tests/test_features.py

```diff
@@ -17,7 +17,7 @@ class Foo(betterproto.Message):
     assert betterproto.serialized_on_wire(foo.bar) is False
 
     # Serialized after setting something
-    foo.bar.baz = 1
+    foo.bar = Bar(baz=1)
```

Comment:
Changing all this is a regression if this PR breaks this

Reply:
I know that isn't great. My PR would break lazy initialization. If you want to keep that, I have to think of another implementation. But lazy initialization also isn't in the google implementation, so I thought it wouldn't be important.

Reply:
Well breaking things isn't good. I know how this can be fixed it's been discussed before but it involves every field having a serialized attribute or similar.